src/UserLabel/UserLabel.tsx

```diff
@@ -7,7 +7,11 @@ import { UserLabelProps } from "./UserLabel.types";
  * Component that renders a user avatar and the name of the user to the right
  * @param label The name of the user to be displayed
  */
-export default function UserLabel({ label, color }: UserLabelProps) {
+export default function UserLabel({
+  label,
+  color,
+  tooltip = ""
```

Comment:
Pass on the optional props to the `IconWithLabel` component and let it handle how it deals with undefined optional props.

```suggestion
  tooltip
```